client/state/jetpack-connect/selectors.js

```diff
@@ -10,4 +10,12 @@ const isCalypsoStartedConnection = function( state, siteSlug ) {
 	return false;
 };
 
-export default { isCalypsoStartedConnection };
+const getFlowType = function( state, site ) {
```

Comment:
First argument of a selector should always be the full state object:

>You'll note in this example that the entire state object is passed to the selector. We've chosen to standardize on always sending the entire state object to any selector as the first parameter. This consistency should alleviate uncertainty in calling selectors, as you can always assume that it'll have a similar argument signature. More importantly, it's not uncommon for selectors to need to traverse different parts of the global state, as in the example above where we pull from both the sites and posts top-level state keys.

https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#selectors